meta-facebook/meta-wedge100/conf/layer.conf

```diff
@@ -8,3 +8,5 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "wedge100"
 BBFILE_PATTERN_wedge100 = "^${LAYERDIR}/"
 BBFILE_PRIORITY_wedge100 = "25"
+
+LAYERSERIES_COMPAT_wedge100 = "warrior"
```

Comment:
Id change this to "warrier rocko"
Since we will continue to build rocko till we fully qualify the platform on the new layer and then deprecate the old.

Reply:
Sure, we can do that, the variable LAYERSERIES_COMPAT_<override> does exist in rocko as well, but the warnings/errors for building without it are not present yet, basically rocko does not require it to be set, but I do agree, we should specify both.